.eslintrc.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
   extends: '@mate-academy/eslint-config',
-  parser: "babel-eslint",
```

Comment:
why did you delete it?

Reply:
That wasn’t me, i didn’t even open this file, should be some mistake here, sorry(